src/testHelpers/apiTestHelpers.ts

```diff
@@ -81,14 +117,20 @@ export async function createUser({
   };
 
   const user = await User.query().insert(data);
-  const userToParticipantRolesData = participantToRoles?.map((item) => ({
-    ...item,
-    userId: user.id,
-    userRoleId: item.userRoleId ?? UserRoleId.Admin,
-  }));
-  if (userToParticipantRolesData) {
-    await UserToParticipantRole.query().insert(userToParticipantRolesData);
```

Comment:
This was causing problems in the SQLite DB when trying to insert multiple rows, e.g. multiple participants for the same user. This is because it relies on batch insert which is only supported by Postgres and MSSQL.